src/modules/chat_deleted_messages/index.js

```diff
@@ -78,6 +81,15 @@ class ChatDeletedMessagesModule {
         tmiRoom.on('clearchat', onClearChat);
         tmiRoom._bttvClearChatMonkeyPatched = true;
     }
+
+    checkDeleted($el, msgObj) {
```

Comment:
1. this breaks emojis
2. turn off deleted links tbh.. there's a boolean somewhere iirc